assets/src/scripts/components/hydrograph/index.js

```diff
@@ -77,34 +128,34 @@ class Hydrograph {
         addSROnlyTable({
             container: this._element,
             columnNames: [this._title, 'Time'],
-            data: this._data.map((value) => {
+            data: this._tsData.current.map((value) => {
                 return [value.value, value.time];
             })
         });
         // We'll actually be appending to a <g> element
-        const plot = svg.append('g')
+        this.plot = this.svg.append('g')
             .attr('transform', `translate(${MARGIN.left},${MARGIN.top})`);
 
         // Create x/y scaling for the full (100%) view.
-        const {xScale, yScale} = createScales(
-            this._data,
+        this.scale = createScales(
+            this._tsData.current,
             WIDTH - MARGIN.right,
             HEIGHT - (MARGIN.top + MARGIN.bottom)
         );
-        const {xAxis, yAxis} = createAxes(xScale, yScale, -WIDTH + MARGIN.right);
+        this.axis = createAxes(this.scale.xScale, this.scale.yScale, -WIDTH + MARGIN.right);
 
         // Draw the graph components with the given scaling.
         appendAxes({
-            plot,
-            xAxis,
-            yAxis,
+            plot: this.plot,
+            xAxis: this.axis.xAxis,
```

Comment:
Still being a D3 neophyte, what's happening to the tick marks on the x-axis? Do they still so labels like "Jan 21, 2018"? Since the `createAxes` function doesn't appear to have changed, it looks like it might. Would that be confusing to a user comparing data with 2017, or would the legend be sufficent?

Reply:
I don't change the x axis and yes it still shows Jan 21, 2018. I think I will change the formatting to not show the year. The two time series are plotted using different xScales. And that's another thing that needs to be done is adding a legend but I think that should be a different ticket at this point.

Reply:
I've vote for having an separate axis label for each date range - could color-code the labels rather than add a legend.

Reply:
One bonus of separate labels is it will continue to work if we do "compare to last week" or similar.

Reply:
So visually would you suggest stacking the labels or putting an axis at the top?

Reply:
An axis on top would be my preference

Reply:
Not sure. I feel like I need to pick up a book on data visualization.

Reply:
I also think adding a legend with year would work if we are always comparing year to year.  But I agree with Dan's point which would also make handling leap years clearer.  So still leaning toward two axes. I'll try coding that up to see how it looks.